### ISSUE TITLE
Not Supported in Laravel 7

### ISSUE THREAD

**Issue by sp1009** [brozot/Laravel-FCM#190]
Hello there,

Can you assure me when you provide this package support on laravel 7? If you provide any time when you provide the support it really helps for me.

Thanks in advance

**Comment by Alexuisis** [comment]
same here

**Comment by yasikrehman** [comment]
Yes, package is not installed in laravel 7.

**Comment by gpibarra** [comment]
Yep...you can see this issue. There are a one solution with other fork, for now, is the only solution.

[https://github.com/brozot/Laravel-FCM/issues/175](https://github.com/brozot/Laravel-FCM/issues/175)

**Comment by gpibarra** [comment]
Waiting for merge.
[https://github.com/brozot/Laravel-FCM/pull/174](https://github.com/brozot/Laravel-FCM/pull/174)
